scripts/ntlm-methods.nse

```diff
@@ -0,0 +1,58 @@
+local http = require "http"
+local shortport = require "shortport"
+local stdnse = require "stdnse"
+local string = require "string"
+local table = require "table"
+
+description = [[
+Returns authentication methods a winrm server supports.
+]]
+
+---
+-- @usage
+-- nmap --script winrm-auth-methods <target>
+--
+-- @output
+-- 5985/tcp open  wsman
+-- | winrm-auth-methods: 
+-- |   Accepted Authentication Methods: 
+-- |     Negotiate
+-- |     Basic
+-- |     Kerberos
+-- |_    CredSSP
+
+author = "Evangelos Deirmentzoglou"
+license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
+categories = {"default", "discovery", "safe"}
+
+portrule = shortport.port_or_service(5985, 5986)
+local data = stdnse.generate_random_string(5)
+
+
+action = function(host, port)
+
+  local r = {}
+  local result = stdnse.output_table()
+  local url = stdnse.get_script_args(SCRIPT_NAME .. '.root') or "/wsman"
```

Comment:
Is the root ever anything but "/wsman"? If not, then no need to have a separate script arg. If so, then this is fine.